plugins/out_azure/azure_conf.c

```diff
@@ -206,18 +207,9 @@ int flb_azure_conf_destroy(struct flb_azure *ctx)
         return -1;
     }
 
-    if (ctx->customer_id) {
-        flb_sds_destroy(ctx->customer_id);
-    }
     if (ctx->dec_shared_key) {
         flb_sds_destroy(ctx->dec_shared_key);
     }
-    if (ctx->shared_key) {
-        flb_sds_destroy(ctx->shared_key);
```

Comment:
This is also buggy. Since the current code allocates a memory block
for this property (via `flb_sds_create()`; see line 52), you need to make
sure to free that block somewhere.